indi-atik/atik_ccd.cpp

```diff
@@ -56,34 +58,47 @@ void ATIK_CCD_ISInit()
         iAvailableCamerasCount = 0;
         std::vector<std::string> cameraNames;
 
-        iAvailableCamerasCount = ArtemisDeviceCount();
-        if (iAvailableCamerasCount > MAX_DEVICES)
-            iAvailableCamerasCount = MAX_DEVICES;
+        IDLog("Atik Cameras API V%d DLL V%d initializing.", ArtemisAPIVersion(), ArtemisDLLVersion());
+
+        for (int loop = 0; loop < MAX_CONNECTION_RETRIES; loop++)
+        {
+            iAvailableCamerasCount = ArtemisDeviceCount();
+
+            if (0 < iAvailableCamerasCount)
+                break;
+
+            IDLog("No Atik Cameras detected on attempt %d/%d, retrying...", loop+1, MAX_CONNECTION_RETRIES);
```

Comment:
You does not increment loop value in this loop anywhere. It should probably be:
`IDLog("No Atik Cameras detected on attempt %d/%d, retrying...", loop++, MAX_CONNECTION_RETRIES);`

Reply:
Er, isn't this part of the "for" construct?

Reply:
You're right ;-) Sorry about this. I was too fast